harness/wellKnownIntrinsicObjects.js

```diff
@@ -8,452 +8,283 @@ defines: [WellKnownIntrinsicObjects]
 
 const WellKnownIntrinsicObjects = [
   {
-    intrinsicName: "%Array%",
-    globalNameOrSource: "Array"
```

Comment:
I don't see any value in renaming these. The names that they have are derived from the column header in the spec.

Reply:
it was less of renaming and more of recreating the whole thing from scratch. i can of course put the names back if you feel strongly about it.